### PR TITLE
But/task 15 hide keyboard show icon on all screens do not come back to search city screen after chosen city

### PR DESCRIPTION
**Pull request by ArtemPolovin**

### CODE REVIEW

File: app/src/main/java/com/example/mytestproject/ui/MainActivity.kt [hunk 1]
```diff
@@ -42,14 +42,25 @@ class MainActivity : AppCompatActivity(){
...
+                else -> View.VISIBLE
             }
         }
     }
 
     override fun onSupportNavigateUp(): Boolean {
         return navController.navigateUp(appBarConfiguration) || super.onSupportNavigateUp()
     }
+
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
```
Comment (startandroid) [inline app/src/main/java/com/example/mytestproject/ui/MainActivity.kt]: This menu will be shown on fragments Second and Third, probably. But it shouldn't.

Comment (ArtemPolovin) [reply app/src/main/java/com/example/mytestproject/ui/MainActivity.kt]: Fixed

File: app/src/main/java/com/example/mytestproject/ui/FragmentSecond.kt [hunk 2]
```diff
@@ -17,4 +18,14 @@ class FragmentSecond : Fragment() {
         return inflater.inflate(R.layout.fragment_second, container, false)
     }
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setHasOptionsMenu(true)
+    }
+
+    override fun onPrepareOptionsMenu(menu: Menu) {
```
Comment (startandroid) [inline app/src/main/java/com/example/mytestproject/ui/FragmentSecond.kt]: We need to do it for every new fragment that we add to app. It is easy to forget to do it.
Probably, it is better to show menu only in weather fragments.  

Also there is an option - to create a separate WeatherFragment that will be a container for three nested weather fragments (today, 3 and 10). It is a bit more complicated, but menu will be shown only once - in this container fragment.

Comment (ArtemPolovin) [reply app/src/main/java/com/example/mytestproject/ui/FragmentSecond.kt]: Fixed

**Review by startandroid — APPROVED** [review]

**Pull request merged**